modules/aruco/misc/python/pattern_generator/.gitignore

```diff
@@ -0,0 +1,10 @@
+__pycache__/
+
+# Visual studio
+*.sdf
```

Comment:
Please exclude `.gitignore` file from the patch.

Python files are already handler by root file.
https://github.com/opencv/opencv_contrib/blob/4.1.1/.gitignore#L2

MSVS (as external tool) should be handled by global `.gitignore` on user machine.

Reply:
No problem

Reply:
@alalek  should I add __pycache__/ to root gitignore?

Reply:
[No](https://github.com/opencv/open_model_zoo/pull/159#discussion_r294365430).